ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -19,10 +19,10 @@ let prototyped_of_field = function
       Some "22.27.0"
   | "host", "last_software_update" ->
       Some "22.20.0"
-  | "pool", "migration_compression" ->
-      Some "22.33.0"
   | "pool", "coordinator_bias" ->
       Some "22.34.0-next"
+  | "pool", "migration_compression" ->
+      Some "22.33.0"
```

Comment:
I have similar in other PR, should this be an issue?

Reply:
rebasing should be easy, as it's the same change, from what I've seen. worst case you can ignore the change and rerun make to regenerate the file